apps/meteor/server/services/federation/infrastructure/matrix/Bridge.ts

```diff
@@ -166,6 +169,29 @@ export class MatrixBridge implements IFederationBridge {
 		}
 	}
 
+	public async verifyInviteeId(externalInviteeId: string): Promise<VerificationStatus> {
+		const [userId, homeserverUrl] = extractUserIdAndHomeserverFromMatrixId(externalInviteeId);
+		try {
+			const response = await fetch(`https://${homeserverUrl}/_matrix/client/v3/register/available?username=${userId}`);
+
+			if (response.status === HttpStatusCodes.BAD_REQUEST) {
```

Comment:
Check if response is ok before using status

Reply:
If I understood, following node-fetch [example](https://github.com/node-fetch/node-fetch#handling-client-and-server-errors), using `response.ok` is the same as `response.status >= 200 && response.status < 300` which in this case will only make a redundant check since the only values we care are `200` and `400`.  

The way we're using the endpoint is inverted in a sense, 400 means VERIFIED for this method and 200 means UNVERIFIED, any other status or error will return UNABLE_TO_VERIFY